### ISSUE TITLE
Add a Create Username page

### ISSUE THREAD

**Issue by kmw101** [iov-one/ponferrada#249]
In the demo wallet the second step after registration is to create a human readable address.

This view will be adapted but the routing to this page from the sign-up is no longer needed.

- [x] Prepare view and storybook (PR #488)
- [ ] Implement register username request logic (PR #505)
- [ ] Add transaction item component for username registration tx
- [ ] E2E tests

**Comment by webmaster128** [comment]
Depends on #305

**Comment by webmaster128** [comment]
Closing in favour of #424 and #405
